mautic/mautic
# Allow the use of = signs in tokens

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅
| New feature/enhancement? (use the a.x branch)      | ✅
| Deprecations?                          | ❌
| BC breaks? (use the c.x branch)        | ❌
| Automated tests included?              |  ❌
| Related user documentation PR URL      |  
| Related developer documentation PR URL |  
| Issue(s) addressed                     | 



#### Description:

This PR adds enhances the regular expression to take "=" characters in account in tokens.

#### Steps to test this PR:


1. Create some tokens which have an '=' sign in the token name
1. Make sure tokens everywhere are being replaced as expected

#### Other areas of Mautic that may be affected by the change:
1. All places where tokens are replaced for emails.

[//]: # ( As applicable: )
#### List of areas covered by the unit and/or functional tests:
